lib/components/src/empty-state/empty-state.ts

```diff
@@ -0,0 +1,79 @@
+import { LitElement, html } from "lit";
+import { customElement, property } from "lit/decorators";
+import styles from './empty-state.scss';
+import '../icon/icon';
+
+declare global {
+  interface HTMLElementTagNameMap {
+   'td-empty-state': CovalentEmptyStateBase;
+  }
+}
+
+@customElement('td-empty-state')
+export class CovalentEmptyStateBase extends LitElement {
+  static override styles = [styles];
+  
+  @property({type: String}) headline = '';
+  @property({type: String}) subtitle = '';
+  @property({type: String}) icon = '';
+  override render() {
+    return html`
+      <style>
+        .content {
+          width: 632px; 
+          height: 300px;
+          display: flex; 
+          flex-direction: column; 
+          align-items: center; 
+          justify-content: center;
+          text-align: center;
+        }
+        .icon-background {
+          display: flex;
+          justify-content: center;
+          align-items: center;
+          width:96px;
+          height:96px;
+          border-radius: 50%;
+          background-color: var(--mdc-theme-background);
+          margin-left: auto;
+          margin-right: auto;
+        }
+        .covalent-icon {
+          font-size: 40px;
+        }
+        .mdc-typography--headline6 {
+          margin: 16px 0 8px 0;
+          font-weight: 600 !important;
+        }
+        .mdc-typography--caption {
+          margin: 0;
+          font-size: 12px;
```

Comment:
you should be able to pull in css variables here for the typography values. for example this on `var(--mdc-typography-caption-font-size)` you can reference the values here lib/components/styles/theme/_all-theme.scss